templates/server.erb

```diff
@@ -13,6 +13,17 @@ remote <%= rem %>
 server-poll-timeout <%= @server_poll_timeout %>
 <% end -%>
 <% end -%>
+<% if @extca_enabled -%>
+ca <%= @extca_ca_cert_file %>
+cert <%= @extca_server_cert_file %>
+key <%= @extca_server_key_file %>
+<%   unless @remote or !@extca_dh_file -%>
+dh <%= @extca_dh_file %>
+<%   end -%>
+<%   unless @remote or !@crl_verify or !@extca_ca_crl_file -%>
+crl-verify <%= @extca_ca_crl_file %>
```

Comment:
There is already a `crl-verify` on https://github.com/luxflux/puppet-openvpn/pull/181/files#diff-802d2d256313233cb337adc27a84b4a4L23. I am pretty sure this will mess up something if it's specified twice.